src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,15 @@
+package core.basesyntax;
+
+public class Truck extends Machine {
+
+    @Override
+    public void doWork() {
+
+        System.out.println("Truck machine started its work");
+    }
+
+    public void stopWork() {
+
+        System.out.println("Truck machine stopped its work");
+    }
+}

```

Comment:
```suggestion
    @Override
    public void doWork() {
        System.out.println("Truck machine started its work");
    }

    public void stopWork() {
        System.out.println("Truck machine stopped its work");
    }
}
```

same, we don't need these empty lines